misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractIterableNotToContainEntriesExpectationsTest.kt

```diff
@@ -0,0 +1,294 @@
+                    )
+                },
+                {
+                    notToContainNullableEntriesSpec.invoke(
+                        this,
+                        { toBeGreaterThan(8.0) },
+                        arrayOf(expectLambda<Double> {})
+                    )
+                }
+            ),
+            groupPrefix = "[nullable Element] "
+        )
+    )
+
+    private fun Expect<Iterable<Double?>>.notToContainNullableFun(
+        a: (Expect<Double>.() -> Unit)?,
+        vararg aX: (Expect<Double>.() -> Unit)?,
+    ) = notToContainNullableEntriesSpec(this, a, aX)
+
+    @TestFactory
+    fun empty_iterable__throws_AssertionError() =
+        nonNullableCases(notToContainEntriesSpec, notToContainNullableEntriesSpec) { notToContainFunArr ->
+            fun Expect<Iterable<Double>>.notToContainFun(
+                a: Expect<Double>.() -> Unit,
+                vararg aX: Expect<Double>.() -> Unit,
+            ) = notToContainFunArr(a, aX)
+
+            expect {
+                expect(setOf<Double>().asIterable()).notToContainFun({ toEqual(4.0) })
+            }.toThrow<AssertionError> {
+                message {
+                    toContainRegex(
+                        "$hasANextElement$separator" +
+                                "$indentRootBulletPoint\\Q$explanatoryBulletPoint\\E$notToContainDescr: $separator" +
+                                "$indentRootBulletPoint$indentListBulletPoint\\Q$listBulletPoint\\E$anElementWhichNeedsDescr: $separator" +
+                                "$indentListBulletPoint$afterExplanatory$toEqualDescr: 4.0.*",
+                        "$hintBulletPoint${
+                            DescriptionIterableLikeExpectation.USE_NOT_TO_HAVE_ELEMENTS_OR_NONE.getDefault()
+                                .format(notToHaveElementsOrNoneFunNameSpec)
+                        }"
+                    )
+                }
+            }
+        }
+
+    @TestFactory
+    fun iterable__happy_case() =
+        nonNullableCases(notToContainEntriesSpec, notToContainNullableEntriesSpec) { notToContainFunArr ->
+            fun Expect<Iterable<Double>>.notToContainFun(
+                a: Expect<Double>.() -> Unit,
+                vararg aX: Expect<Double>.() -> Unit,
+            ) = notToContainFunArr(a, aX)
+            testFactory(notToContainEntriesSpec) {
+                it("$toBeGreaterThanFun(1.0) and $toBeLessThanFun(2.0) does not throw") {
+                    expect(oneToSeven()).notToContainFun({ toBeGreaterThan(1.0); toBeLessThan(2.0) })
+                }
+                it("$toEqualFun(1.1), $toEqualFun(2.2), $toEqualFun(3.3) does not throw") {
+                    expect(oneToSeven()).notToContainFun({ toEqual(1.1) }, { toEqual(2.2) }, { toEqual(3.3) })
+                }
+                it("$toEqualFun(3.3), $toEqualFun(1.1), $toEqualFun(2.2) does not throw") {
+                    ch.tutteli.atrium.api.verbs.internal.expect(oneToSeven())
```

Comment:
replace all occurrences of ch.tutteli.atrium.api.verbs.internal.expect with expect (most likely intellij added the package)